install/kubernetes/helm/istio/files/injection-template.yaml

```diff
@@ -171,6 +171,8 @@ containers:
     valueFrom:
       fieldRef:
         fieldPath: metadata.name
+  - name: ISTIO_META_WORKLOAD_PORTS
+    value: {{ "\"[[ annotation .ObjectMeta `traffic.sidecar.istio.io/includeInboundPorts` (includeInboundPorts .Spec.Containers) ]]\"" }}
```

Comment:
This won't work. We don't have double templating anymore (the `[[`). Look at the injected files and you can see.

Reply:
So you just need to do `{{ annotation .... }}`

Reply:
Oh, thanks. I merge this from my old branch and didn't notice. Done